MainTel/MainTel-Non-standard-Operations/MTM2-Non-standard-Procedures/Power-cycle-MTM2-cabinet.rst

```diff
@@ -0,0 +1,174 @@
+.. Do not remove the comments that describe each section. They are included to provide guidance to contributors.
+.. Do not remove other content provided in the templates, such as a section. Instead, comment out the content and include comments to explain the situation. For example:
+	- If a section within the template is not needed, comment out the section title and label reference. Include a comment explaining why this is not required.
+    - If a file cannot include a title (surrounded by ampersands (#)), comment out the title from the template and include a comment explaining why this is implemented (in addition to applying the ``title`` directive).
+
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: *Jacqueline Seronr*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *Te-Wei Tsai*
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _Templates-Title-of-Procedure:
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+#########################
+Power Cycle MTM2 cabinet
+#########################
+
+
+
+.. _Overview:
+
+Overview
+========
+
+.. This section should provide a brief, top-level description of the procedure's purpose and utilization. Consider including the expected user and when the procedure will be performed.
+
+This procedure mirrors the `Power-Cycle section under M2 handling errors`_ authored by Te-Wei, albeit presented in the format of Observatory Operations Documentation.
+
+.. _`Power-Cycle section under M2 handling errors`: https://ts-m2gui.lsst.io/error-handling/error-handling.html#power-cycle
+
+This procedure is the final option for resolving an MTM2 issue during the night. It's utilized when the M2 GUI consistently displays alarms, and the MTM2 cannot transition to a closed-loop status following the steps outlined in the MTM2 Recovery procedure.
+
+.. need to add link to the corresponding and not yet in the site "MTM2 recovery procedure" page
+.. test that works  
+.. :doc:`MTM2 Recovery </MainTel/MainTel-Non-standard-Operations/MTM2-Non-standard-Procedures/MTM2-Non-standard-Procedures-index>`
+.. The correct webpage would be in M2 Troubleshooting so </MainTel/MainTel-Troubleshooting/MTCS-Troubleshooting/M2-Troubleshooting/MTM2-Recovery>`
+
+.. _Precondition:
```

Comment:
Label here should be unique in the repository for this section, usually "Title-of-the-document-name-of-the-section" , for instance, in this case, something like this "MTM2-Power-Cycle-Cabinet-Precondition"